spec/requests/api/alerts_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe "Alerts API" do
+  let(:alert_definition) { FactoryGirl.create(:miq_alert, :severity => "info") }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+    run_get(alerts_url)
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "reads all alerts statuses" do
```

Comment:
I'm confused by this description - I would try to keep it consistent with the external API ("alerts") even though the back end implementation calls them "alert statuses"